_posts/2017-04-03-chuang-critique.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Critique of Think a Negative Tweet from Trump Crushes a Stock? Think Again"
+date:   2017-04-10
+author: Aileen Chuang
+---
+
+This is a critique of a graphics story from the WSJ.
+
+[Data journalism work from The Wall Street Journal, Feb. 23, 2017](http://www.wsj.com/graphics/trump-market-tweets/).
+
+## Think a Negative Tweet From Trump Crushes a Stock? Think Again
```

Comment:
The headings here are a bit redundant -- you could have just written the critique.